### PR TITLE
Add article about deep linking on iOS

### PR DESCRIPTION
**Pull request by TsvetinaIvanova**

### CODE REVIEW

File: troubleshooting/cannot-open-custom-url-scheme-from-ios-app.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: Deep Linking Not Working on iOS Devices
+description: Opening an external app from an iOS app using a custom URL scheme doesn't work
```
Comment (PreslavKozovski) [inline troubleshooting/cannot-open-custom-url-scheme-from-ios-app.md:3]: does not(following Iva's approach)

File: troubleshooting/cannot-open-custom-url-scheme-from-ios-app.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+position:
+tags: deep linking, custom url scheme, googlegmail, comgooglemaps, iOS
+ticketid: 1062970
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>
```
Comment (PreslavKozovski) [inline troubleshooting/cannot-open-custom-url-scheme-from-ios-app.md]: <td>Code ({{site.ab}})</td>

File: troubleshooting/cannot-open-custom-url-scheme-from-ios-app.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>	
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Run on Device</td>	
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>
```
Comment (PreslavKozovski) [inline troubleshooting/cannot-open-custom-url-scheme-from-ios-app.md]: <td>Hybrid {{site.ac}}</td>

**Comment by PreslavKozovski** [comment]
+1

**Comment by AntonDobrev** [comment]
👍

**Pull request merged**